test/components/DayPickerRangeController_spec.jsx

```diff
@@ -796,6 +796,142 @@ describe('DayPickerRangeController', () => {
         });
       });
 
+      describe('blocked', () => {
+        describe('focusedInput did not change', () => {
+          it('does not call isBlocked', () => {
+            const isBlockedStub =
+              sinon.stub(DayPickerRangeController.prototype, 'isBlocked');
+            const wrapper = shallow(<DayPickerRangeController {...props} />);
+            isBlockedStub.reset();
+            wrapper.instance().componentWillReceiveProps({
+              ...props,
+            });
+            expect(isBlockedStub.callCount).to.equal(0);
+          });
+        });
+
+        describe('focusedInput changed', () => {
+          const visibleDays = {
+            [toISOMonthString(today)]: {
+              [toISODateString(today)]: [],
+              [toISODateString(moment().add(1, 'day'))]: [],
+              [toISODateString(moment().add(2, 'day'))]: [],
+            },
+          };
+          const numVisibleDays = 3;
+
+          it('calls isBlocked for every visible day', () => {
+            const isBlockedStub =
+              sinon.stub(DayPickerRangeController.prototype, 'isBlocked');
+            const wrapper = shallow(<DayPickerRangeController {...props} />);
+            wrapper.setState({ visibleDays });
+            isBlockedStub.reset();
+            wrapper.instance().componentWillReceiveProps({
+              ...props,
+              focusedInput: END_DATE,
+            });
+            expect(isBlockedStub.callCount).to.equal(numVisibleDays);
+          });
+
+          it('if isBlocked(day) is true calls addModifier with `blocked` for each day', () => {
+            const addModifierSpy = sinon.spy(DayPickerRangeController.prototype, 'addModifier');
+            const isBlockedStub =
+              sinon.stub(DayPickerRangeController.prototype, 'isBlocked').returns(true);
+            const wrapper = shallow(<DayPickerRangeController {...props} />);
+            wrapper.setState({ visibleDays });
+            wrapper.instance().componentWillReceiveProps({
+              ...props,
+              focusedInput: START_DATE,
+            });
+            const blockedCalendarCalls = getCallsByModifier(addModifierSpy, 'blocked');
+            expect(blockedCalendarCalls.length).to.equal(numVisibleDays);
+            isBlockedStub.restore();
```

Comment:
You don't need the restore - just don't assign the stub to a variable. In other words, `sinon.stub(foo, bar);` is fine by itself.